docs/sources/flow/getting-started/collect-prometheus-metrics.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Collect Prometheus metrics
+weight: 200
+---
+
+# Collect Prometheus metrics
+
+Grafana Agent Flow can be configured to collect [Prometheus][] metrics and
+forward them to any Prometheus-compatible database.
+
+This topic describes how to:
+
+* Configure metrics delivery
+
+[Prometheus]: https://prometheus.io
+
+## Components used in this topic
+
+* [prometheus.remote_write][]
+* [prometheus.scrape][]
+
+[prometheus.remote_write]: {{< relref "../reference/components/prometheus.remote_write.md" >}}
+[prometheus.scrape]: {{< relref "../reference/components/prometheus.scrape.md" >}}
+
+## Before you begin
+
+* Ensure that you basic familiarity with instrumenting applications with
+  Prometheus.
+* Have a set of Prometheus exports or applications exposing Prometheus metrics
+  that you want to collect metrics from.
+* Identify where you will write collected metrics, such as Prometheus, Grafana
+  Mimir, Grafana Cloud, or Grafana Enterprise Metrics.
+* Be familiar with the concept of [Components][] in Grafana Agent Flow.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+
+## Configure metrics delivery
+
+Before components can collect Prometheus metrics, you must have a component
+responsible for writing those metrics somewhere.
+
+The [prometheus.remote_write][] component is responsible for delivering
+Prometheus metrics to one or Prometheus-compatible endpoints. Once a
+`prometheus.remote_write` component is defined, other Grafana Agent Flow
+components can be used to forward metrics to it.
+
+To configure a `prometheus.remote_write` component for metrics delivery,
+complete the following steps:
+
+1. Add the following `prometheus.remote_write` component to your configuration file:
+
+   ```river
+   prometheus.remote_write "LABEL" {
+     endpoint {
+       url = "PROMETHEUS_URL"
+     }
+   }
+   ```
+
+2. Replace `LABEL` with a label to use for the component, such as `default`.
```

Comment:
It might be more clear if each main point is for adding a component, and there are subpoints for modifying the provided component template. So for example, replacing "LABEL" would be 1.1.
This would make it more consistent with the basic_auth component.

Reply:
I'm not sure that follows the guidelines for steps, but I'm genuinely not sure. I'll merge as-is and we'll get feedback from the docs team afterwards and clean up if we have to.